monitoring/grafana-dashboard.json

```diff
@@ -1654,7 +1654,7 @@
             "uid": "prometheus"
           },
           "editorMode": "code",
-          "expr": "load_dataset_duration_ms{quantile=~\"${percentile}\", pod=~\"${pods}\"} unless load_dataset == 0",
+          "expr": "datasets_acceleration_load_duration_ms{quantile=~\"${percentile}\", pod=~\"${pods}\"} unless load_dataset == 0",
```

Comment:
This metric doesn't exist since OTel doesn't create the summary. I will add this in a follow-on PR.